tritium-tracing/build.gradle

```diff
@@ -4,7 +4,7 @@ dependencies {
 
     api project(':tritium-api')
     api project(':tritium-core')
-    api 'com.palantir.remoting3:tracing'
+    api 'com.palantir.tracing:tracing'
```

Comment:
Probably want this to be split into API & impelementation now:

```
api 'com.palantir.tracing:tracing-api'

implementation 'com.palantir.tracing:tracing'
```

Reply:
added d8c91fe

Reply:
👍